.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish npm package
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    name: Build and publish npm package
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.DEVOLUTIONSBOT_TOKEN }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      
+      - name: Go to wasm directory
+        shell: bash
+        run: cd ./ffi/wasm
+
+      - name: Tests
+        shell: pwsh
+        run: ./run_tests.ps1
+
+      - name: Build & Publish
+        shell: pwsh
+        env:
+          # FIXME: provide a token in env vvvv
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
@Devolutions/devops 
I probably need a new secret in some environment for npm.
The picky npm package will be published to the public npm registry (since picky itself is open source).